### DISCUSSION TITLE
Aleph browser plugin

### DISCUSSION THREAD

**Discussion by pudo** [alephdata/aleph#1521] [Ideas]
* Upload the current page you're viewing to Aleph
* Extract entity profiles from FB, LinkedIn, etc. pages
* Extract corporate networks from OpenCorporates, BvD Orbis
* Do long-term uploads from your local drive (perhaps even a sync folder?)

**Comment by zzzepezzz** [comment]
Is a PoC have been developed yet ? This plugin could be very useful
